render-ws-java-client/src/main/java/org/janelia/render/client/StackAlignmentComparisonClient.java

```diff
@@ -1,44 +1,48 @@
 package org.janelia.render.client;
 
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParametersDelegate;
 import mpicbg.models.AffineModel2D;
 import mpicbg.models.InterpolatedAffineModel2D;
 import mpicbg.models.Model;
 import mpicbg.models.RigidModel2D;
 import mpicbg.models.TranslationModel2D;
-import net.imglib2.util.Pair;
 import org.janelia.alignment.match.CanvasMatches;
+import org.janelia.alignment.match.OrderedCanvasIdPair;
+import org.janelia.alignment.match.OrderedCanvasIdPairWithValue;
 import org.janelia.alignment.spec.ResolvedTileSpecCollection;
 import org.janelia.alignment.spec.TileSpec;
+import org.janelia.alignment.util.FileUtil;
 import org.janelia.render.client.newsolver.solvers.WorkerTools;
-import org.janelia.render.client.solver.SerializableValuePair;
+import org.janelia.render.client.parameter.CommandLineParameters;
+import org.janelia.render.client.parameter.MatchCollectionParameters;
+import org.janelia.render.client.parameter.RenderWebServiceParameters;
 import org.janelia.render.client.solver.StabilizingAffineModel2D;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.File;
-import java.io.FileWriter;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Collection;
-import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 import java.util.function.DoubleBinaryOperator;
 import java.util.stream.Collectors;
 
-// TODO: make this into a full fledged command line tool
-//       * add command line arguments
-//       * break up computation and save into z-layers (could cause problems for larger stacks)
-//       * switch output format to json (see FileUtil.saveJsonFile)
-public class StackAlignmentComparisonClient {
+public class StackAlignmentComparisonClient extends CommandLineParameters {
```

Comment:
Please create nested parameters class that extends CommandLineParameters as is done in [TilePairClient](https://github.com/saalfeldlab/render/blob/ibeam_msem/render-ws-java-client/src/main/java/org/janelia/render/client/TilePairClient.java#L58).